### DISCUSSION TITLE
tutorials/lift-off-part1/06-apollo-explorer

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#15] [course feedback]
# tutorials/lift-off-part1/06-apollo-explorer

https://www.apollographql.com/tutorials/lift-off-part1/06-apollo-explorer

**Comment by matthewsanner** [comment]
When I try to save my operation, it tells me "Could not fetch collections to save to". And when I try to look at Operation Collections it tells me "Error getting collections". I'm signed in to my account. Any ideas?

**Reply by mabuyo** [reply]
Hey @gravityskunk, sorry to hear you're having issues with operation collections. I can't reproduce on my end, would you mind sharing a screenshot of the screen when you're seeing the error? I can check with the team to see how we can troubleshoot this.

**Reply by agreppi** [reply]
I have the same exact issue:

Operation Collections
Saved operations for your Sandbox, across all endpoints
Error getting collections

I didn't find a way to paste a screenshot.

**Reply by mabuyo** [reply]
Hey @agreppi , sorry to hear this! Can you try accessing [https://studio.apollographql.com/sandbox/explorer](https://studio.apollographql.com/sandbox/explorer), then connecting to your server on `http://localhost:4000` by pasting that URL on the top left input? Let me know how that goes.

**Reply by martiniucanastasia** [reply]
I got the same error - `Could not fetch collections to save to`, but I tried to restart server and it started working!

**Reply by rayhatfield** [reply]
I got this too. I clicked the "Leave Sandbox" icon in the top right, which took me to `studio.apollographql.com` and after signing in there I returned to my local server and this problem was gone. (I'd never signed in there before and I suspect it was an account setup or auth/token thing?)

<img width="257" alt="image" src="https://github.com/apollographql-education/content-feedback/assets/1164081/a2f98547-b891-4d37-8158-2f6bd4428c81">

**Reply by ishkabir1401** [reply]
it was helpful , rayhatfield

**Comment by jgerton** [comment]
Not sure why, but I can't save the operation.  I get a pop up dialog that with "Could not fetch collections to save to".  I'm logged in, the operation runs fine.  Is there a sandbox setting that needs to be enabled/disabled?

**Reply by jgerton** [reply]
well, it wasn't in the tutorial, but I opened up apollographql.com site in a new tab and logged in again.  that seemed to be the trick...i think

**Reply by lizhennessy** [reply]
Hmm, that's interesting @jgerton - I'm glad it's working in a new tab. Perhaps there was some difficulty in fetching the default `Sandbox` collection to save the operation to. Please let us know if you encounter any other errors and we'll do our best to look into it!

**Reply by reactivicky** [reply]
I faced the same problem> Not able to save

**Comment by CMH7** [comment]
It did takes me around 5-15 min. just to solve the quiz and my errors are just simple typographies of the words XD.

**Comment by snehal1791** [comment]
I am getting an error while trying to save operations. The error says `Could not fetch collections to save to`. Can someone help?

**Reply by CMH7** [reply]
This is because you opened the Sandbox in your local. Try to open it in Apollo studio then sandbox it should automatically connect to the localhost of your graphql server

**Reply by snehal1791** [reply]
![Screenshot of the error](https://user-images.githubusercontent.com/19614419/231162903-34ac5efc-54e0-435c-b611-24c2935c39fa.png)

**Reply by snehal1791** [reply]
Thank you @CMH7
It worked for me!

**Reply by CMH7** [reply]
Welcome @snehal1791

**Reply by SofiaTohmeGarnero** [reply]
Thank you @CMH7
I had the same problem than @nehal1791 and your solution worked for me too!!

**Comment by marcusj89** [comment]
wu did it

**Comment by satyam9838** [comment]
I am getting response like this -:{
  "data": {
    "tracksForHome": [
      {
        "id": "ec48fb85-3cba-4042-9cd2-0c0c0987fe6e",
        "title": "Hello World",
        "author": {
          "id": "9bb359c9-ae19-4ce5-abb6-d9902d436a17",
          "name": "Hello World",
          "photo": "Hello World"
        },
        "thumbnail": "Hello World",
        "length": -15,
        "modulesCount": -37
      },
      {
        "id": "fee25ef0-a98c-42df-bd76-89befb582b8c",
        "title": "Hello World",
        "author": {
          "id": "1128a10a-9133-4cf0-a0d4-f1131921ce2e",
          "name": "Hello World",
          "photo": "Hello World"
        },
        "thumbnail": "Hello World",
        "length": 73,
        "modulesCount": -18
      }
    ]
  }
}
Why apollo mock data is not being mapped?

**Reply by mabuyo** [reply]
Hey there, can you check your Explorer settings? Click the Gear icon, then make sure "Mock responses" is set to OFF.

If it still doesn't work, happy to look at your server code to troubleshoot there, perhaps the mocks config isn't set properly.

**Reply by zhaoou00** [reply]
I am having the same problem. Even if I do:
	  schema: addMocksToSchema({
	    schema: makeExecutableSchema({ typeDefs }),
	    mocks,
	  }),
It still returns me "Hello World"s

Again, no hints nor error messages. I did copy and paste the "mock.js" from this page with no changes.

**Reply by mabuyo** [reply]
Hi @zhaoou00 - have you gone through the troubleshooting steps I provided? Will copy-paste below:

Can you check your Explorer settings? Click the Gear icon, then make sure "Mock responses" is set to OFF.

If it still doesn't work, happy to look at your server code to troubleshoot there, perhaps the mocks config isn't set properly!

**Reply by zhaoou00** [reply]
yes "Mock responses" is set to OFF.

**Reply by zhaoou00** [reply]
How do I do mocks config?

**Reply by mabuyo** [reply]
That's good, so we can rule that out. I'll need to look at your server code to troubleshoot. Could you push up your code to a repo and share the link with me?

**Reply by mabuyo** [reply]
You can also try comparing your code with the final state: https://github.com/apollographql/odyssey-lift-off-part1/blob/main/final/server/src/index.js

**Reply by zhaoou00** [reply]
Thanks. 
This https://github.com/apollographql/odyssey-lift-off-part1/blob/main/final/server/src/index.js works great. 

However based on this index.js, as soon as I replaced the mocks with:
const mocks = require('./mocks');
The result reverts to "Hello World" again.

Again my mocks.js is directly copied from this page and it is placed in the same directory.

**Reply by mabuyo** [reply]
Hmm, we didn't create a `mocks.js` file, so I'm a little confused where that is coming from. But since it's working with the mocks _inline_ in the same file, it does make me think that the problem might be with the way `mocks` is being exported/imported from that new file you've created.

**Comment by zibiv** [comment]
We defined schema for SpaceCat, then created mock data for SpaceCat in the previous lesson. Why does we need to use query for GetTracks in Code Challenge?))

**Reply by mabuyo** [reply]
Hey there @zibiv! I can see where the confusion is coming from. We created mocks for `SpaceCat` in the previous lessons as code challenges only, for additional practice. We _also_ created mocks for the hands-on project for Tracks data. In this current lesson, we wanted a way for you to validate that everything is working in the hands-on project, which is why we're asking for the results of the `GetTracks` query. Hope that helps clarify!

**Reply by zibiv** [reply]
Thank you for your explanation!

**Comment by marksikaundi** [comment]
this part i have enjoyed

**Comment by meugeniatr** [comment]
Hello, I am receiving the same data as @satyam9838
My server file looks like this:
```
const { addMocksToSchema } = require("@graphql-tools/mock");
const { makeExecutableSchema } = require("@graphql-tools/schema");
const { ApolloServer } = require("@apollo/server");
const { startStandaloneServer } = require("@apollo/server/standalone");
const typeDefs = require("./schema");

const mocks = {
  Query: () => ({
    tracksForHome: () => [...new Array(6)],
  }),
  Track: () => ({
    id: () => "track_01",
    title: () => "Astro Kitty, Space Explorer",
    author: () => {
      return {
        name: "Grumpy Cat",
        photo:
          "https://res.cloudinary.com/dety84pbu/image/upload/v1606816219/kitty-veyron-sm_mctf3c.jpg",
      };
    },
    thumbnail: () =>
      "https://res.cloudinary.com/dety84pbu/image/upload/v1598465568/nebula_cat_djkt9r.jpg",
    length: () => 1210,
    modulesCount: () => 6,
  }),
};

async function startApolloServer() {
  const server = new ApolloServer({
    schema: addMocksToSchema({
      schema: makeExecutableSchema({ typeDefs }),
    }),
    mocks,
  });
  const { url } = await startStandaloneServer(server);

  console.log(`
  🚀  Server is running!
  📭  Query at ${url}
`);
}

startApolloServer();
```

**Reply by meugeniatr** [reply]
I made sure the mocks were off and it still does not work :(

**Reply by mabuyo** [reply]
Hey there @meugeniatr! In your server initialization, `mocks` should be a property for the object argument inside `addMocksToSchema()`, _not_ `new ApolloServer()`.

```js
const server = new ApolloServer({
    schema: addMocksToSchema({
      schema: makeExecutableSchema({ typeDefs }),
      mocks,
    }),
  });
```

You can also compare to the [final state of the code](https://github.com/apollographql/odyssey-lift-off-part1/blob/main/final/server/src/index.js#L28-L33) in the `final` folder! Hope that helps!

**Reply by meugeniatr** [reply]
ohhhhhhhh I see ! thanks a lot!!!! :)

**Reply by Stark-Akshay** [reply]
I am still getting the same hello world error below is my code:


const {ApolloServer} = require("@apollo/server");
const {startStandaloneServer} = require("@apollo/server/standalone");
const typeDefs = require("./schema");
const {addMocksToSchema}  = require("@graphql-tools/mock");
const {makeExecutableSchema} = require("@graphql-tools/schema");

const mocks = {
    Query: () => ({
      tracksForHome: () => [...new Array(6)],
    }),
    Track: () => ({
      id: () => 'track_01',
      title: () => 'Astro Kitty, Space Explorer',
      author: () => {
        return {
            id: 'author_01',
          name: 'Grumpy Cat',
          photo: 'https://res.cloudinary.com/dety84pbu/image/upload/v1606816219/kitty-veyron-sm_mctf3c.jpg',
        };
      },
      thumbnail: () => 'https://res.cloudinary.com/dety84pbu/image/upload/v1598465568/nebula_cat_djkt9r.jpg',
      length: () => 1210,
      modulesCount: () => 6,
    }),
  };

async function startApolloServer() {
    const server = new ApolloServer({
        schema: addMocksToSchema({
          schema: makeExecutableSchema({ typeDefs }),
          mocks,
        }),
      });
    const {url} = await startStandaloneServer(server);

    console.log(`
        Server is running!
        Query at ${url}
    `);
}

startApolloServer();

**Reply by marksikaundi** [reply]
You can try read more on the topic provided and you will be able to fix it, I will review the chapter and share the solution

**Comment by repiemm** [comment]
I was stuck but finally realized I was declaring const server outside the startApolloServer() function 😱

**Comment by talwar** [comment]
I am getting connection refused when I run http://localhost:4000/ on chrome. I did run the following command on my terminal 

saurabhtalwar@Saurabhs-MacBook-Pro server % npx diagnose-endpoint@1.1.0 --endpoint=http://localhost:4000
Diagnosing http://localhost:4000
⚠️  Connection refused (ECONNREFUSED)
Is the address correct?
Is the server running?

**Reply by talwar** [reply]
Problem solved !

**Reply by talwar** [reply]
was not running npm start

**Comment by vajjha** [comment]
talwar I am getting  Connection refused (ECONNREFUSED) too
what did you do resolve it  

apollograph\odyssey-lift-off-part1\server>npm start

> catstronauts-server-complete@1.0.0 start
> npm run compile && nodemon ./dist/index.js

> catstronauts-server-complete@1.0.0 compile
> tsc
[nodemon] 3.1.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,cjs,json
[nodemon] starting `node ./dist/index.js`
\[nodemon] clean exit - waiting for changes before restart

**Reply by lizhennessy** [reply]
Hi there @vajjha ! Could you paste the contents of your server's `src/index.ts` file? It sounds like there might be some configuration missing. Happy to take a look!

**Reply by vajjha** [reply]
Thanks for the prompt response @lizhennessy !!
I caught my error 
when defining const { url }.. I had it outside of aync function startApolloServer()

now all good

**Comment by LingVuDev** [comment]
The code challenge is a little bit unaccurate since you are asking for the first entry from of the apollo explorer query which returns json and your challenge starts with a js const. It does accept the json output, but it should be rephrased to pasting the json instead.

**Comment by NishantCoder108** [comment]
I am not able to figure out , where am i doing wrong in above code challenge?
Can anyone complete this challenge , can anyone help me ?

my response below
```
const firstTrackEntry = {
  // paste the first entry value in the constant here {id:...,}
  Track: () => ({
        id: () => "track_01",
        title: () => "Astro Kitty, Space Explorer",
        author: () => {
            return {
                name: "Grumpy Cat",
                photo: "https://res.cloudinary.com/dety84pbu/image/upload/v1606816219/kitty-veyron-sm_mctf3c.jpg",
            };
        },
        thumbnail: () =>
            "https://res.cloudinary.com/dety84pbu/image/upload/v1598465568/nebula_cat_djkt9r.jpg",
        length: () => 1210,
        modulesCount: () => 6,
    }),
}

**Reply by lizhennessy** [reply]
Hi there @NishantCoder108! In this code challenge, we use the Explorer to run the given query. Here's what that looks like in my Explorer:
<img width="1919" alt="Screenshot 2024-03-05 at 11 39 05" src="https://github.com/apollographql-education/content-feedback/assets/95302380/1b0adff3-a165-4475-8e67-3792ca35e0b5">

So to pass the Code Challenge, we need to copy the contents of the first object (outlined in orange) and paste them into the `firstTrackEntry` object. I hope that helps! Let us know if you have any other questions.

**Reply by NishantCoder108** [reply]
Thank you @lizhennessy

**Comment by azimaro** [comment]
This is what I see when I first go to Apollo Explorer:
query ExampleQuery {
  id
  tracksForHome {
  }
}
As you can see, there is an "id" outside of tracksForHome, and I cannot figure out why. 
Here is my index.ts code:

import { ApolloServer } from "@apollo/server";
import { startStandaloneServer } from "@apollo/server/standalone";
import { addMocksToSchema } from "@graphql-tools/mock";
import { makeExecutableSchema } from "@graphql-tools/schema";
import { typeDefs } from "./schema";

const mocks = {
    Query: () => ({
      tracksForHome: () => [...new Array(6)],
    }),
    Track: () => ({
      id: () => "track_01",
      title: () => "Astro Kitty, Space Explorer",
      author: () => {
        return {
          id: "author_01",
          name: "Grumpy Cat",
          photo: "https://res.cloudinary.com/dety84pbu/image/upload/v1606816219/kitty-veyron-sm_mctf3c.jpg",
        };
      },
      thumbnail: () =>
        "https://res.cloudinary.com/dety84pbu/image/upload/v1598465568/nebula_cat_djkt9r.jpg",
      length: () => 1210,
      modulesCount: () => 6,
    }),
  };

async function startApolloServer() {
    const server = new ApolloServer({
        schema: addMocksToSchema({
          schema: makeExecutableSchema({ typeDefs }),
          mocks,
        }),
    });
    const { url } = await startStandaloneServer(server);
    console.log(`
      🚀  Server is running!
      📭  Query at ${url}
    `);
}

startApolloServer();


Can anyone see the problem? I have been looking for over an hour. :(

**Reply by azimaro** [reply]
Btw, is there a way to include code that keeps the formatting?

**Reply by lizhennessy** [reply]
Hi there @azimaro ! It looks like that might be the default query that shows up in Sandbox before anything is connected. For example, if I just open up Sandbox without a server running, it populates this query by default:

<img width="1910" alt="image" src="https://github.com/apollographql-education/content-feedback/assets/95302380/8017604a-0ea3-4d4e-91fe-116c0d43e75f">

Can you try opening up a new tab in the Explorer and rewriting your query? Your mocks code looks correct to me otherwise!

Also, to answer your question about the code formatting, you can enclose your code within a set of three backticks (```).

It ends up looking like this!

```
hi @azimaro !
```

**Reply by azimaro** [reply]
No, it's not the default, it is connected, the server is running. This is what I see on the left:
Documentation
Root
Query

Query

Fields




tracksForHome:
[Track!]!

**Reply by lizhennessy** [reply]
Could you take a screenshot of what your Explorer looks like?

**Reply by azimaro** [reply]
I took a screenshot, but how do I put it here?

**Reply by azimaro** [reply]
Screenshot 2024-03-26 at 11.07.37 AM

**Reply by azimaro** [reply]
Copy & paste doesn't seem to work

**Reply by lizhennessy** [reply]
Hi @azimaro , that's okay - can you try clicking the plus button at the top of the Operation panel? This opens a new workspace for you to write your query in.

<img width="914" alt="image" src="https://github.com/apollographql-education/content-feedback/assets/95302380/26d21e28-d118-45de-ba91-b11b062d54bb">

Then go to the Documentation panel and click on the _tracksForHome_ field under the _Query_ type. This should add it to the Operation panel, where you can continue building.

**Reply by azimaro** [reply]
Thanks, I built the query without issues, I guess I will just ignore that the id was in the wrong place and move on. :)

**Comment by pvnotpv** [comment]
ah, now graphql is making sense

**Comment by shob101** [comment]
While trying to run server and accessing Apollo explorer through Url "http://localhost:4000/" I am facing empty response issue.
showing "This page isn’t working" with "localhost didn’t send any data."

below pasting my server/src/index.ts file

```ts
import { ApolloServer } from "@apollo/server";
import { addMocksToSchema } from "@graphql-tools/mock";
import { makeExecutableSchema } from "@graphql-tools/schema";
import { startStandaloneServer } from "@apollo/server/standalone";
import { typeDefs } from "./schema";

const mocks = {
  Query: () => ({
    tracksForHome: () => [...new Array(6)],
  }),
  Track: () => ({
    id: () => "track_01",
    title: () => "Astro Kitty, Space Explorer",
    author: () => {
      return {
        name: "Grumpy Cat",
        photo:
          "https://res.cloudinary.com/dety84pbu/image/upload/v1606816219/kitty-veyron-sm_mctf3c.jpg",
      };
    },
    thumbnail: () =>
      "https://res.cloudinary.com/dety84pbu/image/upload/v1598465568/nebula_cat_djkt9r.jpg",
    length: () => 1210,
    modulesCount: () => 6,
  }),
};

async function startApolloServer() {
  const server = new ApolloServer({
    schema: addMocksToSchema({
      schema: makeExecutableSchema({ typeDefs }),
      mocks,
    }),
  });
    const { url } = await startStandaloneServer(server);
    console.log(`
      🚀  Server is running!
      📭  Query at ${url}
    `);
};

startApolloServer();
```

Thank you for any kind of help in advance.

**Reply by mabuyo** [reply]
Hey there @shob101! Is your server running successfully?

**Reply by shob101** [reply]
![server_ss](https://github.com/apollographql-education/content-feedback/assets/57738862/99aa490c-5634-4149-a436-c6627fee0883)

Hey @mabuyo, yes server is running successfully.

Diagnosing http://localhost:4000/
⚠️  Failed to diagnose what went wrong. Here's the error:  RequestError: socket hang up
    at ClientRequest.<anonymous> (C:\Users\TEST\AppData\Local\npm-cache\_npx\44b567a0d18365cd\node_modules\got\dist\source\core\index.js:956:111)  
    at Object.onceWrapper (node:events:626:26)
    at ClientRequest.emit (node:events:523:35)
    at origin.emit (C:\Users\TEST\AppData\Local\npm-cache\_npx\44b567a0d18365cd\node_modules\@szmarczak\http-timer\dist\source\index.js:43:20)      
    at Socket.socketOnEnd (node:_http_client:519:9)
    at Socket.emit (node:events:523:35)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at connResetException (node:internal/errors:717:14)
    at Socket.socketOnEnd (node:_http_client:519:23)
    at Socket.emit (node:events:523:35)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'ECONNRESET',
  timings: {
    start: 1713293360071,
    socket: 1713293360072,
    lookup: 1713293360073,
    connect: 1713293360075,
    secureConnect: undefined,
    upload: 1713293360075,
    response: undefined,
    end: undefined,
    error: 1713293361631,
    abort: undefined,
    phases: {
      wait: 1,
      dns: 1,
      tcp: 2,
      tls: undefined,
      request: 0,
      firstByte: undefined,
      download: undefined,
      total: 1560
    }
  }
} socket hang up

**Reply by mabuyo** [reply]
Thanks! The code snippet looks good to me, so I'm not sure what's going on. One thing I would check is to make sure your code is in the correct file and changes are saved. You can also try navigating to the `final/server` folder and running the server from there, that should be a working version of the project.

Could you also post a screenshot of Explorer where the error is popping up? And if you can push up your repo, I am happy to clone and take a look myself!

**Comment by Mahdyrll** [comment]
Why it says"apollo sandbox cannot be loaded; it appears that you might be offline."?

**Reply by mabuyo** [reply]
Hi @Mahdyrll can you make sure your server is running?

**Reply by Mahdyrll** [reply]
Yeah. I tried again to make sure and nothing changed.

**Reply by mabuyo** [reply]
Hmm, that's strange. Would you be able to add a screenshot here and the full text copy?

**Reply by Mahdyrll** [reply]
yeah sure!
![Screen Shot 2025-02-11 at 9 45 07 PM](https://github.com/user-attachments/assets/d1c456e2-87ba-4b73-8f2d-cfa1f50ecd88)
![Screen Shot 2025-02-11 at 9 45 15 PM](https://github.com/user-attachments/assets/4f138dd6-d3a4-4d83-aa3d-aef53b3a9b33)

**Reply by mabuyo** [reply]
I think there may be something in your settings that's preventing it from loading. Are you connected to the internet through a company VPN? One other thing you can try is going to this URL directly: https://studio.apollographql.com/sandbox/explorer then inputting in `http://localhost:4000` as the URL at the top.

**Reply by Mahdyrll** [reply]
no I'm not using any vpn. checkout my console. one question i have is that is this feature available for all the countries and regions? maybe it's banned in my location.
![Screen Shot 2025-02-11 at 10 15 52 PM](https://github.com/user-attachments/assets/de9680af-43bd-4c52-b51d-1dd40f0982e5)

**Reply by mabuyo** [reply]
Thanks for the screenshot, that's helpful. Is https://studio.apollographql.com/sandbox/explorer blocked for you?

**Reply by Mahdyrll** [reply]
Apparently yes. it shows a completely blank page

**Comment by onlyroz** [comment]
Why do we not specify an id in the author mock e.g.
author: () => {
      return {
       id: "author_01",
        name: "Grumpy Cat",
        photo:
          "https://res.cloudinary.com/apollographql/image/upload/v1730818804/odyssey/lift-off-api/catstrophysicist_bqfh9n_j0amow.jpg",
      };
    },
If I try to do this I get an error. There seems to be a generated author ID which is a guid.  Where does this come from?

**Reply by mabuyo** [reply]
Hi there! What error are you getting? 

But to answer your question, we don't need to specify the author ID. If a property is not specified, the mocks package we imported will take care of generating random data for us.

**Reply by onlyroz** [reply]
Right I think I understand now.  I think originally I was incorrectly using functions in the author mock e.g.
```
author: () => {
  return {
  id: () => "author_01",
  name: "Grumpy Cat",
  photo:
  "https://res.cloudinary.com/apollographql/image/upload/v1730818804/odyssey/lift-off-api/catstrophysicist_bqfh9n_j0amow.jpg",
};
},
```
If I don't do that and just have `id: "author_01"` then this works

**Comment by austinav** [comment]
The Railway deploy doesn't seem to copy/fork the repo as mentioned. They have changed it and there doesn't seem to be an easy way to do it anymore, besides manually copying the repo yourself and then deploying in Railway.

**Comment by salahodinbh** [comment]
Thank you very much for such an easy to follow and interactive guide! Really appreciate it. I just wanted to leave a little feedback, I kind of doubt it's a bug but I couldn't affirm it to being intended either.

When already having a query that looks for some of the properties of an object inside an object, which in this case is the Author object, and then you go to the upper levels that encapsulate it and click on Select all fields recursively, it doesn't appear to add the rest of the fields of the Author object. On the other hand, if the Author object wasn't in the query in the first place, it does add all of them.

So for example, before clicking on that button I already had a query asking for the Author object plus the name. I then went up to the "tracksForHome" level, clicked on Select all fields recursively and added all the fields of that level without making any changes to the Author object of the query.

Other than that, it's a great guide, thank you very much! (Also, sorry for the long read if this was totally intended)

**Comment by tailornp** [comment]
The "Hello World" issue comes in when the key in the schema is not correctly mapped with the key in the mock, and if graphQL mocks finds no mapping it mocks data and for type String it maps "Hello World"

I was able to fix this issue by verifying the key name in the schema (schema.js) and the mock object(index.js).
